src/components/duelling-picklist/picklist-item/picklist-item.component.js

```diff
@@ -15,6 +19,8 @@ export const PicklistItem = React.forwardRef(
       item,
       index,
       handleKeyboardAccessibility,
+      locked,
+      lockedTooltipMessage = "This item is locked and can not be moved",
```

Comment:
I think it might be better naming this something more generic such as `tooltipMessage`, to avoid a breaking change if the functionality is expanded to non-locked items